A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/EICAS/ECAM/A320_Neo_LowerECAM_Engine.js

```diff
@@ -63,6 +63,10 @@ var A320_Neo_LowerECAM_Engine;
                     ignLeftTargetState = A320_Neo_LowerECAM_Engine.Definitions.IGN_STATE.A;
                 }
             }
+            if (n2Percent > 50) {
+                // Close left valve
+                this.engineLeft.setEngineBleedValveState(false, false);
+            }
```

Comment:
This can all be made into one arg, no need for 2 seperate n2Percent.
```suggestion
            if (n2Percent > 50) {
                // Close left & right valve
                this.engineLeft.setEngineBleedValveState(false, false);
                this.engineRight.setEngineBleedValveState(false, false);
            }
```